website/assets/js/codecfuncs.js

```diff
@@ -15,6 +15,15 @@ var metric_units = {
   'encode_cputime': 'Encode CPU time in seconds'
 };
 
+// The GraphLine object is used to get a line and its associated
+// formats together in one piece.
+function GraphLine(title, data, options) {
+  this.title = title;
+  this.data = data;
+  this.options = options;
+}
+
+
 function FillInOneTable(url, id) {
```

Comment:
Yup. GraphLine is actually OK - it's a constructor. But all the other functions should start with a lower case letter. Filed issue #47 to get it right.